webrtc/src/peer_connection/certificate.rs

```diff
@@ -295,18 +227,21 @@ mod test {
         let kp1 = KeyPair::generate(&rcgen::PKCS_ECDSA_P256_SHA256)?;
         let kp1_pem = kp1.serialize_pem();
         let cert1 = RTCCertificate::from_key_pair(kp1)?;
-        let cert1_pem = cert1.pem();
+        let cert1_pem = pem(&cert1);
 
         let kp2 = KeyPair::generate(&rcgen::PKCS_ECDSA_P256_SHA256)?;
-        let _cert2 = RTCCertificate::from_key_pair(kp2)?;
+        let cert2 = RTCCertificate::from_key_pair(kp2)?;
 
         let kp3 = KeyPair::from_pem(kp1_pem.as_str())?;
         let kp3_pem = kp3.serialize_pem();
-        let _cert3 = RTCCertificate::from_pem(cert1_pem, kp3)?;
+        let cert3 = RTCCertificate::from_pem(&cert1_pem, kp3)?;
 
         assert_eq!(kp1_pem, kp3_pem);
-        //assert!(cert1 != cert2);
-        //TODO: assert!(cert1 == cert3);
+        assert_ne!(cert1, cert2);
+        // TODO: assert!(cert1 == cert3);
```

Comment:
this TODO is also super weird. these certificates can't be equal because we're using `cert1` as a CA to create `cert3`.

Reply:
fixed